components/x-content-block/src/ContentBlock.jsx

```diff
@@ -1,12 +1,29 @@
 import { h } from '@financial-times/x-engine';
+import ShareButtons from './ShareButtons';
+import Timestamp from './Timestamp';
 import styles from './ContentBlock.scss';
 
-const ContentBlock = (props) => (
-	<div className='x-content-block'>
-		<h1 className={styles.heading}>{props.title}</h1>
-		{props.isBreakingNews && <span>Breaking News!!!</span>}
-		<p dangerouslySetInnerHTML={{__html: props.content}} />
-	</div>
-);
+const ContentBlock = (props) => {
+	const {
+		postId,
+		title,
+		content,
+		publishedTimestamp,
+		isBreakingNews,
+		articleUrl
+	} = props;
+
+	return (
+		<article className={styles['content-block']} id={`post-${postId}`} data-trackable="live-post">
```

Comment:
Do you use this `id` for anything? The reason I ask is that I've added an `id` to the container that wraps `<article>` in `next-article` because it wasn't able to access this one. If you don't need it, I think we can just delete this?

Reply:
Thanks Keran! Removed that.